test/controllers/DefermentControllerSpec.scala

```diff
@@ -25,7 +25,7 @@ import models.requests.{DataRequest, IdentifierRequest, OptionalDataRequest}
 import models.underpayments.UnderpaymentDetail
 import models.{UserAnswers, UserType}
 import pages.underpayments.UnderpaymentDetailSummaryPage
-import pages.{CheckModePage, DefermentPage, UserTypePage}
+import pages.{CheckModePage, DefermentPage, SplitPaymentPage, UserTypePage}
```

Comment:
If theres more than 3 imports should use ._, usually auto code formatting cleans it up, there's more than one import like this in the file